ios-target/View/Home/ModalViewController.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  TargetForm.swift
+//  ios-target
+//
+//  Created by sol manini on 6/17/19.
+//  Copyright © 2019 TopTier labs. All rights reserved.
+//
+
+import UIKit
+
+class ModalViewController: UIViewController {
+  
+  lazy var blackView: UIView = {
+    let view = UIView()
+    view.translatesAutoresizingMaskIntoConstraints = false
+    view.alpha = 0
+    return view
+  }()
+  
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    view.insertSubview(blackView, at: 0)
+    
+    NSLayoutConstraint.activate([
+      view.leadingAnchor.constraint(equalTo: blackView.leadingAnchor),
+      view.trailingAnchor.constraint(equalTo: blackView.trailingAnchor),
```

Comment:
you should be getting console warnings with these constraints since there are not enough. If you're setting height and width constraints you should use centerY and X instead of leading and trailing. makes sense ?